facebook/delphiFacebook/R/contingency_variables.R

```diff
@@ -329,82 +332,836 @@ remap_responses <- function(df) {
+  # # fever, along with cough or sore throat / # any B2 response
+  # defined elsewhere and copied here with the required name
+  if ("n_hh_prop_ili" %in% names(df)) {
+    df$n_ili <- df$n_hh_prop_ili
+  } else {
+    df$n_ili <- NA_real_
+  }
+
+  # hh_cmnty_cli
+  # Percentage reporting illness in their local community including their
+  # household
+  # As in API
+  # made elsewhere and renamed in `rename_responses`
+
+  # direct_contact
+  # Percentage of respondents that have reported having had direct contact 
+  # (longer than 1 minute) with people not staying with them.
+  # "respondent = someone who answered any of the four contact types
+  # (responses to at least one contact type > 0) / # (responses to at least one
+  # contact type)"
+  if (all(c("n_contact_num_work", "n_contact_num_shopping",
+            "n_contact_num_social", "n_contact_num_other") %in% names(df))) {
+    df$b_direct_contact <- as.numeric(any_true(
+      df$n_contact_num_work > 0,
+      df$n_contact_num_shopping > 0,
+      df$n_contact_num_social > 0,
+      df$n_contact_num_other > 0
+    ))
+  } else {
+    df$b_direct_contact <- NA_real_
+  }
+
+  # anosmia
+  # Percentage of respondents experiencing anosmia
+  # loss of taste or smell / # any B2 response
+  if ("ms_symptoms" %in% names(df)) {
+    symptoms <- split_options(df$ms_symptoms)
+    df$b_anosmia <- as.numeric(is_selected(symptoms, "13"))
+  } else {
+    df$b_anosmia <- NA_real_
+  }
+
+  # vaccinated
+  # Percentage vaccinated
+  # # yes / # V1 responses
+  # made elsewhere and renamed in `rename_responses`
+
+  # received_2_vaccine_doses
+  # Percentage receiving two doses
+  # # 2 doses/ # V2 responses
+  # made elsewhere and renamed in `rename_responses`
+
+  # accept_vaccine
+  # Percentage who would definitely or probably choose to get vaccinated
+  # # (yes, definitely OR yes, probably) / #V3 responses
+  # made elsewhere and renamed in `rename_responses`
+
+  # hesitant_vaccine
+  # Percentage who would definitely or probably NOT choose to get vaccinated
+  # # (no, definitely not OR no, probably not) / #V3 responses
+  if ("mc_accept_cov_vaccine" %in% names(df)) {	
+    df$b_hesitant_vaccine <- as.numeric(
+      df$mc_accept_cov_vaccine == "prob not vaccinate" | 
+      df$mc_accept_cov_vaccine == "def not vaccinate"
+    )
+  } else {	
+    df$b_hesitant_vaccine <- NA_real_
+  }
+
+  # vaccinated_or_accept
+  # Percentage who have either already received a COVID vaccine or would 
+  # definitely or probably choose to get vaccinated, if a vaccine were offered 
+  # to them today.
+  # # (yes to V1) OR ((yes, definitely OR yes, probably) to V3) / 
+  # # (respondents to V3 OR (yes to V1))
+  if (all(c("b_vaccinated", "b_accept_vaccine") %in% names(df))) {
+    df$b_vaccinated_or_accept <- as.numeric(
+      (!is.na(df$b_vaccinated) & df$b_vaccinated == 1) | df$b_accept_vaccine == 1
     )
   } else {
-    df$b_65_or_older <- NA_real_
+    df$b_vaccinated_or_accept <- NA_real_
   }
```

Comment:
Already defined in `variables.R` as `v_covid_vaccinated_or_accept`. Should just be able to rename it.

Reply:
Done.